portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,17 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random quote to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function addQuote() {
+    const quote =
+        ['“Do You Like Scary Movies?” - Scream', '“Do You Want To Play A Game?” - Saw', '“It Rubs The Lotion On Its Skin Or Else It Gets The Hose Again.” - The Silence Of The Lambs',
+        '"Groovy" - Evil Dead II', '“Nobody trusts anybody now…and we\'re all very tired.” - The Thing', '“It\'s Halloween; I guess everyone\'s entitled to one good scare.” - Halloween'];
```

Comment:
[4.4](https://google.github.io/styleguide/jsguide.html#formatting-column-limit): JavaScript code has a column limit of 80 characters.

I suggest placing each value on a separate line like in [this example](https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma).